cmd/phlare/main.go

```diff
@@ -43,7 +43,12 @@ func (mf *mainFlags) RegisterFlags(fs *flag.FlagSet) {
 	fs.BoolVar(&mf.PrintHelp, "help", false, "Print basic help.")
 	fs.BoolVar(&mf.PrintHelpAll, "help-all", false, "Print help, also including advanced and experimental parameters.")
 }
-
+func errorHandler(testMode bool) {
```

Comment:
Nit: I find moving testMode a little more elegant

```suggestion
func errorHandler() {
	testMode := cfg.GetTestMode()
```

Reply:
yes, looks good, made the changes!